common/db/testdata/test-client-pkcs8-encrypted-demo.pem

```diff
@@ -1,91 +0,0 @@
------BEGIN CERTIFICATE-----
-MIIGYTCCBEmgAwIBAgIJAL+8WDUncZFZMA0GCSqGSIb3DQEBCwUAMIGaMQswCQYD
```

Comment:
Accidentally added this file a while back for some personal testing; removing it now.